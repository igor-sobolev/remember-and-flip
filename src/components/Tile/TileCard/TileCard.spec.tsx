import React from 'react';

import { TileCard } from '.';
import { shallow, ShallowWrapper } from 'enzyme';

const givenProps = {
  className: 'test',
  options: {
    width: 100,
    height: 100,
    margin: 0
  }
};

describe('Given TileCard component', () => {
  describe('when it is rendered', () => {
    let wrapper: ShallowWrapper;
    beforeEach(() => {
      wrapper = shallow(<TileCard {...givenProps} />);
    });

    it('should match the snapshot', () => {
      expect(wrapper.getElement()).toMatchSnapshot();
    });
  });
});
