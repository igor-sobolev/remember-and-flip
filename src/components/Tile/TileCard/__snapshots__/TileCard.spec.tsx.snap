// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given TileCard component when it is rendered should match the snapshot 1`] = `
<div
  className="test"
  style={
    Object {
      "height": 100,
      "margin": 0,
      "width": 100,
    }
  }
/>
`;
